Git.hub/Client.cs

```diff
@@ -148,6 +148,20 @@ public User getCurrentUser()
             return user.Data;
         }
 
+        public User getUser(string username)
```

Comment:
The method must be in PascalCase (e.g. `GetUser`) and arguments must be in camelCase (e.g. `userName`)
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions

Reply:
I would usually agree, but everything else in this class follows camelCase, so I did it too... I can change it if you want, but then it will be inconsistent. I'll fix the param name, though.

Reply:
I haven't actually looked at the existing code at all. 
CamelCase is soooo Javaish and not C#ish

Reply:
Yeah, I know... But the current code is like this. So, which is worse? Using camelCase, or having inconsistent naming? I don't mind either way, just let me know ;)

Reply:
I've looked through the codebase and for the most part it looks to adhere to naming standards. This class is an odd ball. 
So if you would please align you code with the MS naming standards.
We can get the rest fixed separately.

Reply:
👍